setup.py

```diff
@@ -99,7 +99,8 @@ def checkProgramIsInstalled(self, program, args, where_to_download,
                        "pyBigWig >=0.3.7",
                        "hicexplorer >= 2.1.1",
                        "pysam>=0.14",
-                       "pytest"
+                       "pytest",
+                       "gffutils"  # I do not know which version
                        ]
 
 if sys.version_info[0] == 2 or (sys.version_info[0] == 3 and sys.version_info[1] == 4):
```

Comment:
Not required anymore.

Reply:
@joachimwolff feel free to commit directly to the branch.